tests/test_metadata_serialization.py

```diff
@@ -360,6 +360,43 @@ def test_delegation_serialization(self, test_case_data: str):
         delegation = Delegations.from_dict(copy.deepcopy(case_dict))
         self.assertDictEqual(case_dict, delegation.to_dict())
 
+    valid_delegations: utils.DataSet = {
+        "using root as delegate role name": '{"keys": { \
+                "keyid1" : {"keytype": "rsa", "scheme": "rsassa-pss-sha256", "keyval": {"public": "foo"}}}, \
+            "roles": [ \
+                {"keyids": ["keyid"], "name": "root", "terminating": true, "paths": ["fn1"], "threshold": 3}] \
```

Comment:
```suggestion
                {"keyids": ["keyid1"], "name": "root", "terminating": true, "paths": ["fn1"], "threshold": 3}] \
```

Please note that the `keyids` used in `Targets.Delegations.DelegationRole.keys` should then be used inside `Targets.Delegations.DelegationRole.roles`.
See an example in our hardcoded targets file [here](https://github.com/theupdateframework/python-tuf/blob/develop/tests/repository_data/repository/metadata/targets.json).

Please make that change in the other test cases as well.

Reply:
I see I messed up it.